plugins/pipelines/test/lib/hooks_test.rb

```diff
@@ -0,0 +1,42 @@
+require_relative '../test_helper'
+
+describe 'zendesk hooks' do
+  let(:deploy) { deploys(:succeeded_test) }
+  let(:next_deploy) { deploys(:succeeded_production_test) }
+  let(:stage) { deploy.stage }
+  let(:next_stages) { [ stages(:test_production), stages(:test_production_pod) ] }
+  let(:output) { StringIO.new }
+  let(:job) do
+    Job.create(
+      project: stage.project,
+      command: "echo hello world",
+      status: "running",
+      user: User.first,
+      deploy: deploy)
+  end
+
+  describe :after_job_execution do
+    it 'kicks off the next stages in the deploy' do
+      stage.update!(next_stage_ids: next_stages.map(&:id))
```

Comment:
Hmm... tests 1 & 2 need that line, but tests 3 & 4 can't have it.  So it won't be reducing the number of lines to move this into the before.